src/dbnode/persist/fs/commitlog/commit_log_conc_test.go

```diff
@@ -109,7 +109,7 @@ func TestCommitLogActiveLogsConcurrency(t *testing.T) {
 			if err != nil {
 				panic(err)
 			}
-			require.Equal(t, 1, len(logs))
+			require.Equal(t, 2, len(logs))
```

Comment:
code change looks good, but it seems that there are no new tests being added, other than asserting the number of logs/files is 2 (or incrementing by 2). 

Do you think there are any new tests can be added?

Reply:
I couldn't really think of any. The existing tests are already pretty good in that they have a good mix of unit tests, prop tests, and concurrency tests. The change I'm making here is more of an implementation detail than anything so if everything is still passing I think we're in good shape